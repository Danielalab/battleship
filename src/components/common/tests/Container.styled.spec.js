import 'jest-styled-components';
import { render } from '@testing-library/react';
import Container from '../Container.styled';

describe('Container', () => {
  test('Should renders Container component', () => {
    const { container } = render(<Container />);
    expect(container).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });

  test('Should renders Container component with flex-direction & justify-content values', () => {
    const { container } = render(<Container direction="column" justifyContent="center" />);
    expect(container).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
