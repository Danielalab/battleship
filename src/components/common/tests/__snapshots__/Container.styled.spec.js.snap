// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Should renders Container component 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<div>
  <section
    class="c0"
  />
</div>
`;

exports[`Container Should renders Container component with flex-direction & justify-content values 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

<div>
  <section
    class="c0"
    direction="column"
  />
</div>
`;
